frontend/src/pages/cans/detail/CanDetail.jsx

```diff
@@ -25,7 +25,7 @@ const CanDetail = () => {
     return (
         <App breadCrumbName={`${can.description}`}>
             <h1>
-                {can?.number} ({can?.nickname})
+                {can?.number} ({can?.nick_name})
```

Comment:
would this be the same as `display_name`?

Reply:
No the `display_name` is the `can.number`